packages/app/components/stack-modal/StackModal.tsx

```diff
@@ -74,17 +85,32 @@ export const StackModal = ({
   const lastSlot = orderSlots[orderSlots.length - 1];
   const nextSlot = orderSlots[totalOrderSlotsDone(stackOrder)];
 
-  const remainingFunds =
-    convertedAmount(stackOrder.amount, stackOrder.sellToken.decimals) -
-    totalFundsUsed(stackOrder);
+  const stackRemainingFundsWithTokenText = `${stackRemainingFunds(
+    stackOrder
+  )} ${stackOrder.sellToken.symbol}`;
 
-  const remainingFundsDescription =
-    remainingFunds > 0 &&
-    `remaining funds, ${remainingFunds} ${stackOrder.sellToken.symbol}`;
+  const remainingFundsText = `The ${stackRemainingFundsWithTokenText} will be sent to your wallet.`;
 
-  const stackIsComplete =
-    totalOrderSlotsDone(stackOrder) === orderSlots.length &&
-    remainingFunds === 0;
+  const getConfirmCancelContent = (): Content => {
```

Comment:
**Minor, non-blocking**:
`description` property can be abstracted into its own variable as it's the same value

Reply:
It could but don't think it will be much better. 

I like the fact if we want to change anything about the dialog it's all here, otherwise we'll be changing the title on this object and for the description we'll have to go to the component. 

I'll let it be here for now.